WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -21,6 +21,7 @@ class JetpackWindowManager: WindowManager {
 
         guard AccountHelper.isLoggedIn else {
             shouldImportMigrationData ? importAndShowMigrationContent(blog) : showSignInUI()
+            self.migrationTacker.trackContentImportEligibility(eligible: shouldImportMigrationData)
```

Comment:
I think tracking import eligibility should be called before importing and showing the migration UI.

Reply:
Yep, good catch. Fixed in 2d49153fbda47ec182767cc6f165447b60183aed.